README.md

```diff
@@ -45,11 +49,26 @@ environment variable to include the jar which includes all dependencies:
 BF_CP=target/OMEZarrReader-with-dependencies.jar showinf -nopix your.ome.zarr/.zattrs
 ```
 
-## Known Issues/TODO list
-- Currently working on packaging, discovered issue when connecting to S3 using packaged jar
-- S3 File System Store is likely not ideal sceanrio, other options to be investigated
-- S3 access currently very inefficient
-- Odd issue with data being lost when decompressing bytes in jzarr, an ugly hack is currently in place
-- Identification of S3 location needs updating
-- Refactor code to remove duplication
-- Parse colours for labels
+## Reader specific options
+
+The OMEZarrReader has a number of reader specific options in version 0.4.0 which can be used to customise the reader behaviour. This options can be used in the same manner as the reader options for Bio-Formats outlined [here](https://bio-formats.readthedocs.io/en/latest/formats/options.html#usage). 
+
+The new default behaviour of the `omezarr.include_labels` option introduced in v0.4.0 represents a change in behaviour from the v0.3 releases. Previously any Zarr arrays found in the labels folder would by default be represented as an additional image series. With the current default settings, Zarr arrays in the labels folder will no longer be included in the list of image series. Changing this setting to `true` will revert to the previous behaviour.
```

Comment:
If a user has previously imported images with labels using older ZarrReader, I guess that the images will break once they update ZarrReader?
This is probably quite uncommon - maybe we help users deal with this on a case-by-case basis, but maybe a warning of this could help?
I think the fix will involve adding bfoptions to the Fileset as I did at https://github.com/IDR/idr-metadata/issues/684#issuecomment-1885258819 - which is a bit painful psql etc. Any other options to workaround this issue?

Reply:
Yeah, certainly in the OMERO case, if files had been imported with and older version then they would either need to reimport or use the bfoptions. Realistically I don't know of anyone other than IDR that would have been impacted here, but if needs be we could assist with psql for adding bfoptions to the fileset. I have added an extra warning for this scanrio.